packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -2861,6 +2922,28 @@ function getDefaultBranchName() {
 	}
 }
 
+/**
+ * Mock setter for usage within test blocks for dashboard script
+ */
+export function setMockFetchDashScript(mockResponse: string) {
+	msw.use(
+		rest.get(
+			`*/accounts/:accountId/workers/services/:fromDashScriptName/environments/:environment/content`,
+			(req, res, ctx) => {
+				// This is a fake FormData object, until we can get MSW (beta) that supports FormData
+				const mockForm = new FormData();
+				mockForm.append("name", mockResponse);
+				// Sending it as JSON will result in empty object, but cause the fetchDashboardScript to execute its code
+				return res(ctx.status(200), ctx.json(mockForm));
+			}
+		)
+	);
+	// Mock the actual return value that FormData would return
+	(fetchDashboardScript as jest.Mock).mockImplementationOnce(() => {
+		return mockResponse;
+	});
+}
```

Comment:
Ah ok I see, this has to do with MSW formdata support. Can we contribute this similar to how we did with undici?

Reply:
They have Fetch API support in the works, which I captured in this issue: https://github.com/cloudflare/wrangler2/issues/2419

Reply:
Cool